pkg/proto/backup_schedule_service.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package ydbcp;
+option go_package = "github.com/ydb-platform/ydbcp/pkg/proto;ydbcp";
+
+import "backup_schedule.proto";
+import "google/protobuf/duration.proto";
+
+service BackupScheduleService {
+    rpc CreateBackupSchedule (CreateBackupScheduleRequest) returns (BackupSchedule);
+    rpc UpdateBackupSchedule (UpdateBackupScheduleRequest) returns (BackupSchedule);
```

Comment:
in old API, there is no Create\Update, only Backup (which basically means SetBackupScheduleSettingsForDatabase)

Reply:
I settled on leaving only `rpc SetBackupSchedule`